rules/windows/lateral_movement_execution_from_tsclient_mup.toml

```diff
@@ -0,0 +1,40 @@
+[metadata]
+creation_date = "2020/11/11"
+maturity = "production"
+updated_date = "2020/11/11"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies execution from the Remote Desktop Protocol (RDP) shared mountpoint tsclient on the target host. This may indicate lateral movement
+attempt.
```

Comment:
```suggestion
Identifies execution from the Remote Desktop Protocol (RDP) shared mountpoint tsclient on the target host. This may indicate a lateral movement
attempt.
```

Reply:
may need re-lint